custom/panel_templates/Default/core/emails_test.tpl

```diff
@@ -54,6 +53,16 @@
                             <a href="{$SEND_LINK}" class="btn btn-primary">{$SEND}</a>
                             {/if}
 
+                            {if isset($SUCCESS)}
+                            <div class ="email-suggest">
```

Comment:
how does this look styling wise?
also, can we remove the "class="email-suggest"" and use builtin classes instead